### DISCUSSION TITLE
Would you recommend using this admin panel for a large project?

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#1550] [Help]
Can this admin panel work with large databases? I understand it depends on how you write the code or how you write the queries. For example, 15,000 products, 14,000 users, etc... And it is constantly growing. Almost everyone says that Livewire is for small projects.

**Comment by ryangjchandler** [comment]
@ArtMin96 There's a lot dependent on how you write your code. You could quite easily test performance by seeding some records.

**Reply by danielbidala** [reply]
@ArtMin96 15.000 record in a db table is not large. It's rather quite small :) Livewire has nothing to do with the size of your db. Just don’t query thousands of records at once.
